controllers/operator/openstack_controller.go

```diff
@@ -340,6 +354,53 @@ func (r *OpenStackReconciler) countDeployments(ctx context.Context, instance *op
 	return count, nil
 }
 
+func isWebhookEndpoint(name string) bool {
+	endpointNames := []string{"openstack-operator-webhook-service", "infra-operator-webhook-service", "openstack-baremetal-operator-webhook-service"}
+	for _, prefix := range endpointNames {
+		if strings.HasPrefix(name, prefix) {
+			return true
+		}
+	}
+	return false
+}
+
+// checkServiceEndpoints -
+func (r *OpenStackReconciler) checkServiceEndpoints(ctx context.Context, instance *operatorv1beta1.OpenStack) (ctrl.Result, error) {
+
+	// NOTE: this is a static list for all operators with webhooks enabled
```

Comment:
Should we move this comment to just above L358?

Reply:
ack, yes I can move it. I hate to wait for CI just for this :/